### PR TITLE
python3Packages.tinycss2: fix failing tests

### PR DESCRIPTION
**Pull request by misuzu**
###### Motivation for this change
https://hydra.nixos.org/build/122651822

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/tinycss2/default.nix [hunk 1]
```diff
@@ -12,7 +12,14 @@ buildPythonPackage rec {
     sha256 = "1kw84y09lggji4krkc58jyhsfj31w8npwhznr7lf19d0zbix09v4";
   };
 
-  patches = [ ./remove-redundant-dependency.patch ];
+  patches = [
+    ./remove-redundant-dependency.patch
+    # remove this patch on next version bump
```
Comment (jonringer) [inline pkgs/development/python-modules/tinycss2/default.nix:17]: I usually like to know why a patch is being applied. You already did the other half (saying when it's ready to remove) :+1: 
```suggestion
    # fixes tests, remove this patch on next version bump
```

**Review by jonringer — COMMENTED** [review]
otherwise LGTM

File: pkgs/development/python-modules/tinycss2/default.nix (hunk 1 above)
Comment (misuzu) [reply pkgs/development/python-modules/tinycss2/default.nix]: Fixed!

**Comment by balsoft** [comment]
@GrahamcOfBorg build python3Packages.tinycss2 python3Packages.cssselect2

**Comment by balsoft** [comment]
Some tests fail, but the build still succeeds ^

**Comment by misuzu** [comment]
> Some tests fail, but the build still succeeds ^

This is fine: https://travis-ci.org/github/Kozea/cssselect2/jobs/663252998

**Pull request closed without merging**